modules/claims_api/app/controllers/concerns/claims_api/json_format_validation.rb

```diff
@@ -20,13 +19,8 @@ def validate_json_format
       end
 
       def render_body_is_not_a_hash_error(body)
```

Comment:
is this method used elsewhere? might be worth moving this implementation up into line 10 and getting rid of the method.
if it is reused, or having the method is just preferred, maybe we can rename it to something that doesnt start with "render"? since its not actually rendering anything any more.